pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -531,33 +564,31 @@ func (s *Server) createIstioRA(opts *caOptions) (ra.RegistrationAuthority, error
 		}
 
 		// File does not exist.
+		// TODO(costin): panic instead ?
```

Comment:
We donot encourage panic everywhere

Reply:
Agree, there are very few good reasons to panic.

Starting a mesh control plane without proper CA is probably one of the few reasons for panic - it will prevent users from deploying a revision of Istiod ( or upgrading istiod ) with settings  that would bring down the mesh, or to deploy a new Istiod that can't secure. It is a TODO - we can discuss it again when/if it ever gets implemented, but we fail to start Istiod if other parameters are invalid.

Reply:
Do we need a release note for this change in behavior? With this change we are no longer are using the `defaultCACertPath`. Could there be users relying on this dependency?

Reply:
Added a release note already. What do you mean by not using defaultCACertPath ?  We haven't been using it for some time and I think there were release notes when the k8s signer was removed by John, I'm just cleaning leftovers. That CA is the K8S APIserver signer, and we no longer have ability to sign our certs with the K8S CA.

Reply:
I.e. I'm just cleaning up the leftovers.